core/templates/dev/head/profile/profile.html

```diff
@@ -17,37 +17,33 @@
 
 {% block content %}
   <div class="container">
+    
+    <div class="col-xs-12" style="height:50px;"></div>
```

Comment:
This line is needed to add a break between the top, breadcrumbs header and the picture.